### PR TITLE
Update to GHC 8.10.7

### PR DESCRIPTION
**Pull request by tmcgilchrist**

### CODE REVIEW

**Review by newhoggy — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Comment by angerman** [comment]
bors try

**Comment by angerman** [comment]
bors merge

**Comment by angerman** [comment]
bors merge

File: cardano-prelude/README.md [hunk 1]
```diff
@@ -26,6 +26,3 @@ This repo uses `brittany` to enforce a consistent formatting style.
 
 If you have `brittany` installed, the `scripts/brittany/brittany-all-hs` script
```
Comment (Jimbo4350) [inline cardano-prelude/README.md:27]: Should this comment also be removed?

Comment (newhoggy) [reply cardano-prelude/README.md]: Removing!

**Comment by tmcgilchrist** [comment]
bors merge

**Pull request merged**